api/src/main/java/org/apache/apex/api/EmbeddedAppLauncher.java

```diff
@@ -33,25 +32,25 @@
   /**
    * Parameter to specify the time after which the application will be shutdown; pass 0 to run indefinitely.
    */
-  public static final Attribute<Long> RUN_MILLIS = new Attribute<Long>(0L);
+  public static final Attribute<Long> RUN_MILLIS = new Attribute<>(0L);
 
   /**
    * Parameter to launch application asynchronously and return from launch immediately.
    */
-  public static final Attribute<Boolean> RUN_ASYNC = new Attribute<Boolean>(false);
+  public static final Attribute<Boolean> RUN_ASYNC = new Attribute<>(false);
 
   /**
    * Parameter to enable or disable heartbeat monitoring.
    */
-  public static final Attribute<Boolean> HEARTBEAT_MONITORING = new Attribute<Boolean>(true);
+  public static final Attribute<Boolean> HEARTBEAT_MONITORING = new Attribute<>(true);
 
   /**
    * Parameter to serialize DAG before launch.
    */
-  public static final Attribute<Boolean> SERIALIZE_DAG = new Attribute<Boolean>(false);
+  public static final Attribute<Boolean> SERIALIZE_DAG = new Attribute<>(false);
 
   static {
-    Attribute.AttributeMap.AttributeInitializer.initialize(LocalMode.class);
+    Attribute.AttributeMap.AttributeInitializer.initialize(EmbeddedAppLauncher.class);
```

Comment:
Good catch, must have happened when I flipped the inheritance order towards the end of the review.